scanpy/preprocessing/_highly_variable_genes.py

```diff
@@ -62,6 +62,7 @@ def _highly_variable_genes_seurat_v3(
             "`flavor='seurat_v3'` expects raw count data, but non-integers were found.",
             UserWarning,
         )
+    mean_fulldataset, var_fulldataset = _get_mean_var(X)
```

Comment:
Could remove the variables if they're not used anywhere else:

```python
df["mean"], df["var"] =  _get_mean_var(X)
```

Reply:
good point, thanks!